app/views/tags/edit.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="box wide">
+  <div class="legend">
+    Tag Edit
+  </div>
+
+  <div class="boxline">
+    <%= link_to "Filter By Tag #{@tag.tag}", tag_path(@tag) %>
+  </div>
+
+  <%= form_for @tag, url: update_tag_path, method: :post do |f| %>
+    <div class="boxline">
+      <%= f.label :tag, 'Name:', class: 'required' %>
+      <%= f.text_field :tag  %>
+    </div>
+
+    <div class="boxline">
+      <%= f.label :description, 'Description:' %>
+      <%= f.text_field :description %>
+    </div>
+
+    <div class="boxline">
```

Comment:
These three booleans could use a little descriptive help text about what they do.

Reply:
I agree, and perhaps a suggested range for `hotness_mod` as well. Speaking of which, I don't really know what they do right now either, besides `is_media`. I can try and look things up and figure it out, but can anyone chime in on what the intent of these flags is and how `hotness_mod` works?

Reply:
* `hotness_mod` is a multiplier on score in rankings. `1` is normal. Lower values reduce score, higher raises it. Normal range is probably `0.5` to `1.5`.
 * `is_media` tags the medium, nice for noting pdf or videos that some people don't like to see. A story must have at least one non-media tag.
 * `privileged` only moderators can apply this tag, useful for `announce` and other staff threads
 * `inactive` can't be applied to new articles. Useful so you don't delete history if you decide to change your tagging scheme.

Reply:
On the `hotness_mod`, I got concerned when I noticed that all of the tag `hotness_mod`s on my site were `0.0`, so I checked the JSON output on production Lobsters and the code. It appears that `0.0` is normal, and a couple of tags have `-0.25`, so I'll put those ranges in the text.